CubeTest/CubeTest.csproj

```diff
@@ -2,21 +2,22 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFramework>net7.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
         <ImplicitUsings>enable</ImplicitUsings>
         <Nullable>enable</Nullable>
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="BenchmarkDotNet" Version="0.13.5" />
-        <PackageReference Include="Silk.NET.Input.Glfw" Version="2.17.1" />
-        <PackageReference Include="Silk.NET.Input.Sdl" Version="2.17.1" />
-        <PackageReference Include="Silk.NET.WebGPU" Version="2.17.1" />
-        <PackageReference Include="Silk.NET.WebGPU.Extensions.Disposal" Version="2.17.1" />
-        <PackageReference Include="Silk.NET.Windowing.Glfw" Version="2.17.1" />
-        <PackageReference Include="Silk.NET.Windowing.Sdl" Version="2.17.1" />
-        <PackageReference Include="SixLabors.ImageSharp" Version="3.0.1" />
+        <PackageReference Include="BenchmarkDotNet" Version="0.13.10" />
+        <PackageReference Include="Silk.NET.Input.Glfw" Version="2.19.0" />
+        <PackageReference Include="Silk.NET.Input.Sdl" Version="2.19.0" />
+        <PackageReference Include="Silk.NET.WebGPU" Version="2.19.0" />
+        <PackageReference Include="Silk.NET.WebGPU.Extensions.Disposal" Version="2.19.0" />
```

Comment:
i deprecated the `Silk.NET.WebGPU.Extensions.Disposal` package, you should remove this and just use the direct disposal methods on the `WebGPU` object instead